modules/katamari/src/test/ts/atomic/api/str/EmptyTest.ts

```diff
@@ -1,30 +1,33 @@
-import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
 import fc from 'fast-check';
 import * as Strings from 'ephox/katamari/api/Strings';
 
-UnitTest.test('isEmpty: unit test', () => {
-  const check = (expected, str) => {
-    Assert.eq('isEmpty', expected, Strings.isEmpty(str));
-  };
+describe('atomic.katamari.api.str.EmptyTest', () => {
+  it('isEmpty unit test', () => {
+    const check = (expected, str) => {
```

Comment:
```suggestion
    const check = (expected: boolean, str: string) => {
```
Again, not related to the BDD conversion so feel free to ignore.